main.js

```diff
@@ -0,0 +1,160 @@
+class Board {
+  constructor(){
+    this.board =
+    [[1,2,3],
+    [4,5,6],
+    [7,8,9]];
+    this.available=[1,2,3,4,5,6,7,8,9];
+    this.ready = false;
+  }
+  addPlayer(player){
+    if(this.player1===undefined){
+      this.player1=player;
+      this.player1.setTurn(true);
+    }
+    else if(this.player2===undefined){
+      this.player2=player;
+      this.player2.setTurn(false);
+      this.ready = true;
+      this.print();
+    }
+    else{
+      console.log("May only have two players");
+    }
+  }
+  get checkWin(){
+    for(var i=0;i<3;i++){
+      if(this.board[0][i]==this.board[1][i] && this.board[1][i]==this.board[2][i]){
```

Comment:
Tons of hard-coded stuff in here. How could you extract the logic into something cleaner and less prone to bugs?